profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_dgt_connector/tmgmt_dgt_connector_cart/src/Metadata/CartBundleMetadataController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Drupal\tmgmt_dgt_connector_cart\Metadata;
+
+use EntityDefaultMetadataController;
+
+/**
+ * DGT FTT Translator mapping entity.
+ */
+class CartBundleMetadataController extends EntityDefaultMetadataController {
+
+  /**
+   * Override entity properties to add custom ones.
+   */
+  public function entityPropertyInfo() {
+    $info = parent::entityPropertyInfo();
+    $properties = &$info[$this->type]['properties'];
+    $properties['uid'] = array(
+      'label' => t('User ID'),
+      'schema field' => 'uid',
+      'getter callback' => 'entity_property_getter_method',
+      'setter callback' => 'entity_property_verbatim_set',
+      'required' => TRUE,
+      'description' => t('ID of the author.'),
+    );
+    $properties['tjid'] = array(
+      'label' => t('Translation job ID'),
+      'schema field' => 'tjid',
+      'getter callback' => 'entity_property_getter_method',
+      'setter callback' => 'entity_property_verbatim_set',
+      'required' => FALSE,
+      'description' => t('Related translation job ID.'),
+    );
+    $properties['target_languages'] = array(
+      'label' => t('Target languages'),
+      'schema field' => 'target_languages',
+      'getter callback' => 'entity_property_getter_method',
+      'setter callback' => 'entity_property_verbatim_set',
+      'required' => TRUE,
+      'description' => t('Target languages of the bundle.'),
+    );
+    $properties['status'] = array(
```

Comment:
I also had this discussion with @netlooker and we decided to go for readability. 
One of the reasons behind it is that for the last part of the year we have had plenty of issues with DGT and tmgmt requests going wrong and having to be able to quickly assess the status of a request is something that could help with future debugging,
In any case, I'm open to discussion since I'm in favour of changing it to integer :)

Reply:
This PR should be merged ASAP. As this question is not blocking, let's proceed with what we have. We may change in the future, if we manage to convince @netlooker :)

Reply:
:+1: